python/setup.py

```diff
@@ -1,22 +1,18 @@
-import os.path
+import os
 import platform
 
 from setuptools import Extension
 from setuptools import setup
 from setuptools.command.build_ext import build_ext
 
-
 IS_WINDOWS = platform.system() == "Windows"
 
 
-# Obscure magic required to allow numpy be used as a 'setup_requires'.
-# Based on https://stackoverflow.com/questions/19919905
 class local_build_ext(build_ext):
```

Comment:
Do we need this at all any more?

Reply:
Good spot, removed in 03e40911

Reply:
Hmm, removing this worked locally, but not on CI.